Document anonymous access and limitations (#101)
hassio-addons/addon-grafana: grafana/DOCS.md

1. Hit Save & Test

## Anonymous Access

> Anonymous access on a local network is possible by exposing the port in the
> add-on config, along with setting the following environment variables:

Reviewer: ```suggestion
Anonymous access on a local network is possible by exposing the port in the
add-on configuration, along with setting the following environment variables:
```

1. Hit Save & Test

## Anonymous Access

Anonymous access on a local network is possible by exposing the port in the

> addon config, along with setting the following environment variables:

Reviewer: ```suggestion
add-on config, along with setting the following environment variables:
```

```yaml
env_vars:
  - name: GF_AUTH_ANONYMOUS_ACCESS_ENABLED
    value: "true"
  - name: GF_AUTH_ANONYMOUS_ACCESS_ORG_NAME
    value: "Main Org."
  - name: GF_AUTH_ANONYMOUS_ACCESS_ORG_ROLE
    value: "Viewer"
```

It is not possible to enable anonymous or non-administrator access with Home
Assistant Cloud. This includes embedding Grafana resources with an iframe or

> rendered image inside of a dashboard. For more details see

Reviewer: ```suggestion
rendered image inside of a dashboard. For more details see:
```

> ["“401: Unauthorized” iframe CARD of Grafana not working"](https://community.home-assistant.io/t/401-unauthorized-iframe-card-of-grafana-not-working/148017)
> and
> [Anonymous login not working, Grafan add-on 3.0.0 #55](https://github.com/hassio-addons/addon-grafana/issues/55).

Reviewer: ```suggestion

- ["“401: Unauthorized” iframe CARD of Grafana not working"](https://community.home-assistant.io/t/401-unauthorized-iframe-card-of-grafana-not-working/148017)
and
- [Anonymous login not working, Grafana add-on 3.0.0 #55](https://github.com/hassio-addons/addon-grafana/issues/55).
```
